R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
Right, this is correct if both midnight observations are supplied. I think it's more common that the user would supply only one midnight observation, thus we should assume that freq=nobs. Or if we assume that it's equally as likely that the user would supply both midnight obs as they would supply one midnight obs, then the assumption of freq=nobs will be wrong half the time, and I would argue easier to interpret if it's left as freq=nobs.